docs/Release-Notes-2.16.md

```diff
@@ -0,0 +1,134 @@
+* `rocky+epel-8-*` - This uses Rocky Linux 8 + Fedora EPEL. It works right off the bat and it is recommended replacement. The only disadvantage is that Koji actually uses RHEL + EPEL for EPEL builds.
+
+There were [several options how to handle EOL of epel-* configs](https://docs.google.com/document/d/1wF7-7_y6Ac_oB-kCFdE6VBWPW8o8zjXd2Z0SGy4VxUA/edit?usp=sharing) and we asked EPEL Steering Committee to decide and they [decided to remove epel-* config](https://pagure.io/epel/issue/133#comment-765381). So it is up to you to decide what to use.
+
+Mock will ease it and if you try to build for `epel-8-*` config and this config does not exist, you will get this message:
+
+```
+$ mock -r epel-8-x86_64 --shell
+ERROR: Could not find required config file: /etc/mock/epel-8-x86_64.cfg
+ERROR: There are those alternatives:
+ERROR: 
+ERROR: [1] alma+epel-8-x86_64
+ERROR:     Use instead: mock -r alma+epel-8-x86_64 --shell 
+ERROR:     Builds against AlmaLinux 8 repositories, together with the official EPEL repositories.
+ERROR:     Project page: https://almalinux.org/
+ERROR:     Enable permanently by:
+ERROR:     $ ln -s /etc/mock/alma+epel-8-x86_64.cfg /home/praiskup/.config/mock/epel-8-x86_64.cfg
+ERROR: 
+ERROR: [2] centos+epel-8-x86_64
+ERROR:     Use instead: mock -r centos+epel-8-x86_64 --shell 
+ERROR:     Builds against CentOS Linux 8 repositories, together with the official EPEL repositories.
+ERROR:     This will reach end-of-life in January 2021.
+ERROR:     Enable permanently by:
+ERROR:     $ ln -s /etc/mock/centos+epel-8-x86_64.cfg /home/praiskup/.config/mock/epel-8-x86_64.cfg
+ERROR: 
+ERROR: [3] rhel+epel-8-x86_64
+ERROR:     Use instead: mock -r rhel+epel-8-x86_64 --shell 
+ERROR:     Builds against Red Hat Enterprise Linux 8 repositories, together with the official EPEL repositories.
+ERROR:     This mimics what is done in the official EPEL build system, but you need a Red Hat subscription:
+ERROR:     https://rpm-software-management.github.io/mock/Feature-rhelchroots
+ERROR:     Enable permanently by:
+ERROR:     $ ln -s /etc/mock/rhel+epel-8-x86_64.cfg /home/praiskup/.config/mock/epel-8-x86_64.cfg
+ERROR: 
+ERROR: [4] rocky+epel-8-x86_64
+ERROR:     Use instead: mock -r rocky+epel-8-x86_64 --shell 
+ERROR:     Builds against Rocky Linux 8 repositories, together with the official EPEL repositories.
+ERROR:     Project page: https://rockylinux.org/
+ERROR:     Enable permanently by:
+ERROR:     $ ln -s /etc/mock/rocky+epel-8-x86_64.cfg /home/praiskup/.config/mock/epel-8-x86_64.cfg
+```
+
+Additional changes are:
+
+ * Fedora 33 configs were moved to eol/ directory
+ * EOLed EPEL Playground configs
+ * Added configs for CentOS Stream 9 + EPEL Next 9
+ * Added configs for epel8-playground
```

Comment:
I don't think we added `epel8-playground` this cycle, did we? Do we mean that we added EPEL 9 configs here?

Reply:
Ahh, my mistake. Removed.